flexget/components/sites/sites/newznab.py

```diff
@@ -34,32 +36,38 @@ class Newznab:
                 'type': 'string',
                 'enum': ['movie', 'tvsearch', 'tv', 'music', 'book', 'all'],
             },
-            'url': {'type': 'string', 'format': 'url'},
-            'website': {'type': 'string', 'format': 'url'},
```

Comment:
I think better to keep the old properties, then add the oneOf up a level like:
```python
oneOf: [{'required': 'url'}, {'required': 'website'}]
```

Reply:
Good idea. Didn't know you could do that.